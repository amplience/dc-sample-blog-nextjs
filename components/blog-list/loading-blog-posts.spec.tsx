/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import LoadingBlogPosts from './loading-blog-posts';

describe('LoadingBlogPosts', () => {
  it('should render component', async () => {
    const component = await renderer.create(<LoadingBlogPosts/>);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
