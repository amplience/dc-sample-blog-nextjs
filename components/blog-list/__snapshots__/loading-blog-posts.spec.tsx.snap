// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingBlogPosts should render component 1`] = `
<div
  className="jsx-2874207879 loading-blog-posts"
>
  <p
    className="jsx-2874207879"
  >
    Loading...
  </p>
</div>
`;
